### PR TITLE
Use package version

### PR DESCRIPTION
**Pull request by thornbill**
**Changes**
* Pulls the version number used in apphost from package.json
* Fixes warning in apphost

**Issues**
N/A

### CODE REVIEW

**Review by dmitrylyzo — COMMENTED** [review]
It might be easier to use `{ version as appVersion }`.

**Comment by dmitrylyzo** [comment]
The only place with `options` I found.
https://github.com/jellyfin/jellyfin-web/blob/52543f8a515452c9bb4310a9d7275329b8cf887d/src/plugins/htmlVideoPlayer/plugin.js#L1424

**Review by dmitrylyzo — APPROVED** [review]

**Review by Camc314 — APPROVED** [review]

File: src/components/apphost.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+import { version as appVersion } from '../../package.json';
```
Comment (ferferga) [inline src/components/apphost.js:1]:
```suggestion
import { version as appVersion } from '~/package.json';
```
Not tested.

**Review by ferferga — CHANGES_REQUESTED** [review]

File: src/components/apphost.js (hunk 1 above)
Comment (thornbill) [reply src/components/apphost.js:1]: That doesn't work :slightly_smiling_face: 

```
ERROR in ./components/apphost.js 9:0-56
Module not found: Error: Can't resolve '~/package.json' in '/home/bill/Projects/jellyfin-web/src/components'
 @ ./scripts/site.js 33:0-48 144:9-25 207:6-22 212:7-23
```

**Review by ferferga — APPROVED** [review]

**Pull request merged**